src/utils.ts

```diff
@@ -550,7 +550,7 @@ export async function estimateDefaultBridgeDepositL2Gas(
     // due to storage slot aggregation, the gas estimation will depend on the address
     // and so estimation for the zero address may be smaller than for the sender.
     from ??= ethers.Wallet.createRandom().address;
-    if (isBaseTokenDeposit || token === ETH_ADDRESS) {
```

Comment:
Changed the estimation here mostly,` token === Eth_address` was there incorrectly. Also the :
```
const name = l1TokenAddress == ETH_ADDRESS_IN_CONTRACTS ? "Ether" :  await token.name();
    const symbol =  l1TokenAddress == ETH_ADDRESS_IN_CONTRACTS ? "ETH" : await token.symbol();
    const decimals =  l1TokenAddress == ETH_ADDRESS_IN_CONTRACTS ? 18 :  await token.decimals();
```

Reply:
@Raid5594 , please close if ok